mono_repo/lib/src/commands/list_command.dart

```diff
@@ -96,13 +96,21 @@ Iterable<String> listPackages(
   required bool onlyPublished,
   required Set<Column> showItems,
 }) sync* {
-  for (var pkg in rootConfig) {
-    if (onlyPublished && !_published(pkg.pubspec)) {
-      continue;
-    }
+  for (var pkg in enumeratePackages(rootConfig, onlyPublished: onlyPublished)) {
     yield showItems.map((e) => e.valueFor(pkg)).join(',');
   }
 }
 
-bool _published(Pubspec pubspec) =>
-    pubspec.version != null && pubspec.publishTo != 'none';
+Iterable<PackageConfig> enumeratePackages(
+  RootConfig rootConfig, {
+  required bool onlyPublished,
+}) =>
+    rootConfig.where((element) => !onlyPublished || element.pubspec._published);
+
+extension PubspecExtension on Pubspec {
+  bool get _published => version != null && publishTo != 'none';
+
+  String get pubBadge => _published
```

Comment:
[optional] Consider moving to the file where it used.

Reply:
But then I'd need to create another extension...or just a static thing. 🤷 

I hear ya...

Reply:
Are you worried about the 2 lines of overhead? They could both be private unnamed extensions. You could come close to saving characters...

Reply:
Don't smile at him mocking me, @jakemac53 !!

Reply:
okay. Fixing